integrated_channels/cornerstone/transmitters/content_metadata.py

```diff
@@ -15,68 +15,3 @@ def __init__(self, enterprise_configuration, client=CornerstoneAPIClient):
             enterprise_configuration=enterprise_configuration,
             client=client
         )
-
-    def transmit(self, create_payload, update_payload, delete_payload, content_updated_mapping, **kwargs):
```

Comment:
Cornerstone pulls content via an API call. In order to accomplish this in the past, it overrode a lot of the `Transmitter`code and added custom methods. I've tweaked the base implementation signature of `transmit` to return the lists of items it worked on (now that we save all content items). This allows us to call the normal `transmit` and work on the results rather than implement an alternative like we did in the past.